.pipelines/Common.yml

```diff
@@ -224,7 +224,7 @@ stages:
       displayName: Copy language server for packaging
       inputs:
         sourceFolder: '$(Pipeline.Workspace)/drop_build_bicep_windows/bicep.LangServer/'
-        contents: '*.*'
+        contents: '**'
```

Comment:
![bdfeb4b4b29e8d77f55d980125cc6133](https://user-images.githubusercontent.com/16367959/112892395-19f25600-908e-11eb-9070-fab4e6aa4a25.png)

Reply:
Yeah - this took so long to figure out and it ended up being a one-char fix ☹.